agent_plugin_builder/vendor_dirs.py

```diff
@@ -25,15 +25,13 @@
         *LINUX_IMAGE_PYENV_INIT_COMMANDS,
         "cd /plugin",
         "pip install --dry-run -r requirements.txt --report {filename}",
-        "chown {uid}:{gid} {filename}",
     ]
 )
 LINUX_BUILD_VENDOR_DIR_COMMANDS: Final = " && ".join(
     [
         *LINUX_IMAGE_PYENV_INIT_COMMANDS,
         "cd /plugin",
         "pip install -r requirements.txt -t {vendor_path}",
-        "chown -R {uid}:{gid} {vendor_path}",
```

Comment:
Since we're not able to set the user's uid/gid when running the linux container, the files will be written with whatever the container user's uid and gid are, which are currently `root`. How does this affect our plugins when they're built? We're not preserving permissions when we tar, so perhaps the only potential issue is not being able to recurse into directories when building the tar

Reply:
Now we are setting them